scripts/setup_test.py

```diff
@@ -338,6 +365,68 @@ def mock_getuid():
                 'https://github.com/yarnpkg/yarn/releases/download/'
                 'v1.17.3/yarn-v1.17.3.tar.gz'])
 
+    def test_package_install_with_windows_x64(self):
+        def mock_exists(unused_path):
+            return False
+        def mock_getuid():
+            return 10000
+
+        check_function_calls = [
+            'create_directory_is_called',
+            'test_python_version_is_called',
+            'rename_is_called',
+            'delete_file_is_called'
+        ]
+        os_swap = self.swap(common, 'OS_NAME', 'Windows')
+        architecture_swap = self.swap(common, 'ARCHITECTURE', 'AMD64')
+        exists_swap = self.swap(os.path, 'exists', mock_exists)
+        getuid_swap = self.swap(os, 'getuid', mock_getuid)
+
+        with self.test_py_swap, self.create_swap, exists_swap:
+            with self.download_swap, self.rename_swap:
+                with self.delete_swap, self.isfile_swap:
+                    with os_swap, getuid_swap, architecture_swap:
+                        setup.main(args=[])
+        for item in check_function_calls:
+            self.assertTrue(self.check_function_calls[item])
+        self.assertEqual(
+            self.urls, [
+                'https://nodejs.org/dist/v10.15.3/node-v10.15.3'
```

Comment:
Please also make a comment in the setup script to change this string if the node version is upgraded.

Reply:
I made the node version and yarn version as global variables and share it with `setup_test.py`, this should solve the problem.